File: common-security/src/main/java/com/onmeet/common/security/JwtConstants.java
package com.onmeet.common.security;

public final class JwtConstants {
    public static final String AUTHORIZATION_HEADER = "Authorization";
    public static final String BEARER_PREFIX = "Bearer ";
    public static final String ACCESS_TOKEN_COOKIE_NAME = "accessToken";
    public static final String REFRESH_TOKEN_COOKIE_NAME = "refreshToken";
    public static final String USER_ID_CLAIM = "userId";
    public static final String ROLE_CLAIM = "role";

    private JwtConstants() {
        // Private constructor to prevent instantiation
    }
}

File: onmeet-common/src/main/kotlin/com/onmeet/common/security/JwtConstants.kt
package com.onmeet.common.security

object JwtConstants {
    const val AUTHORIZATION_HEADER = "Authorization"
    const val BEARER_PREFIX = "Bearer "
    const val ACCESS_TOKEN_COOKIE_NAME = "accessToken"
    const val REFRESH_TOKEN_COOKIE_NAME = "refreshToken"
    const val USER_ID_CLAIM = "userId"
    const val ROLE_CLAIM = "role"
    const val CLAIM_AUTHORITIES = "authorities"
    const val MEETING_ID_CLAIM = "meetingId"
}
